catalogs/catalog-hadoop/src/main/java/org/apache/gravitino/catalog/hadoop/authentication/kerberos/KerberosClient.java

```diff
@@ -66,7 +65,6 @@ public String login(String keytabFilePath) throws IOException {
 
     // Login
     UserGroupInformation.setConfiguration(hadoopConf);
-    KerberosName.resetDefaultRealm();
```

Comment:
Why do we remove this line here?

Reply:
Please refer to that PR.

This PR is in the background.

https://github.com/apache/gravitino/pull/4304